op-service/eth/account_proof.go

```diff
@@ -51,6 +51,9 @@ func (res *AccountResult) Verify(stateRoot common.Hash) error {
 		if err != nil {
 			return fmt.Errorf("failed to verify storage value %d with key %s (path %x) in storage trie %s: %w", i, entry.Key, path, res.StorageHash, err)
 		}
+		if val == nil && entry.Value.ToInt().Cmp(common.Big0) == 0 { // empty storage is zero by default
```

Comment:
This is useful for proving storage against empty slots?

Reply:
yes, zeroed protocol versions were causing issues here, this handles the zero-default-value case. (it's mentioned in the PR description changes too).